src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/OutputModule.cs

```diff
@@ -60,7 +60,7 @@ internal static string GetFilepathWithExtension(string outputFile, string extens
             internal static string TryGetDirectoryPath(string directory)
             {
                 if (directory == null || directory.Length == 0)
-                    return ".\\";
+                    return string.Concat(@".", Path.DirectorySeparatorChar);
                 else
```

Comment:
Remove the string.Concat here too.

Reply:
@huanwu would you able to make this change ?

Reply:
@kant2002 yes, just commited a new version.